static/app/views/replays/detail/issueList.tsx

```diff
@@ -25,27 +26,30 @@ type Props = {
 const columns = [t('Issue'), t('Graph'), t('Events'), t('Users')];
 
 function IssueList({projectId, replayId}: Props) {
+  const api = useApi();
   const organization = useOrganization();
   const isScreenLarge = useMedia(`(min-width: ${theme.breakpoints.large})`);
 
+  const url = `/organizations/${organization.slug}/issues/`;
+  const query = {
+    query: `replayId:${replayId}`,
+  };
+
   const {
     data: issues = [],
     isLoading,
     isError,
     error,
-  } = useApiQuery<Group[], RequestError>(
-    [
-      `/organizations/${organization.slug}/issues/`,
-      {
-        query: {
-          query: `replayId:${replayId}`,
+  } = useQuery<Group[], RequestError>({
```

Comment:
does `useApiQuery` not support custom headers?

Reply:
no, it does not - we introduced the `header` option for the base api `Client` class

we can add support for it in `useApiQuery` but this feels like a one-off